2_process/src/functions.R

```diff
@@ -0,0 +1,25 @@
+process <- function(input){
+  
+  # Set output directory
+  project_output_dir <- "2_process/out"
+  
+  # Prepare the data for plotting
+  eval_data <- readr::read_csv(input, col_types = 'iccd') %>%
+    filter(str_detect(exper_id, 'similar_[0-9]+')) %>%
+    mutate(col = case_when(
+      model_type == 'pb' ~ '#1b9e77',
+      model_type == 'dl' ~'#d95f02',
+      model_type == 'pgdl' ~ '#7570b3'
```

Comment:
I recommend pulling out the color values and making it a vector argument that can be modified as needed.